source/default/_patterns/00-protons/tailwind.tokens.css

```diff
@@ -1,7 +1,14 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
-
+/**
+ * Use this directive to control where Tailwind injects the responsive
+ * variations of each utility.
+ *
+ * If omitted, Tailwind will append these classes to the very end of
+ * your stylesheet by default.
+ */
+@tailwind screens;
```

Comment:
@brittrich upss. I forgot how important that is. Certainly, a benefit that I'm working on frontend related tasks.
I added an example on this PR on how to use the responsive breakpoints with TailwindCSS. Eating my own dog food for sure hahaha.

Reply:
This won't solve the issue you notice with YML files but it will make those media screen points available on twig. It took me a while to figure out why my media queries were death.